src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -3694,6 +3927,28 @@ type TicketType implements Node {
     zones: [String!]
 }
 
+input TransferPreferences {
+    costPenalty: Int
+    minimumWaitingTime: Duration
+}
+
+input TransitModePreferences {
+    mode: TransitMode!
+    reluctance: CostFunction
+}
+
+input TransitPreferences {
+    """TODO should we reuse this or make a new design"""
+    unpreferred: InputUnpreferred
```

Comment:
Did we have support for preferrring certain things already or was it still unimplemented? I'm not sure why I have omitted it from here.